app/models/team.rb

```diff
@@ -53,6 +53,10 @@ def by_season_phase
         Team.visible.where season: Season.current
       end
     end
+
+    def active_teams
+      Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
```

Comment:
I would rename the scope to `current_selected` or something like that. I wouldn't duplicate the term "team" in the scope since we're already telling `Team` to return teams.

Another idea: maybe modularize the scopes so that there is a `current` one (with all teams for the current season) and `selected` with (`kind: %w(sponsored voluntary)`). The required scope can then be composed with `current.selected` (and the other scopes can be refactored/DRYed to re-use the new scopes).